spec/models/shelter_spec.rb

```diff
@@ -1,6 +1,23 @@
 require "rails_helper"
 
 RSpec.describe Shelter, type: :model do
+  let!(:shelter_1) {Shelter.create!(name: "Aurora shelter", city: "Aurora, CO", foster_program: false, rank: 9)}
+  let!(:shelter_2) {Shelter.create!(name: "RGV animal shelter", city: "Harlingen, TX", foster_program: false, rank: 5)}
+  let!(:shelter_3) {Shelter.create!(name: "Fancy pets of Colorado", city: "Denver, CO", foster_program: true, rank: 10)}
+
+  let!(:pet_1) {shelter_1.pets.create!(name: "Mr. Pirate", breed: "tuxedo shorthair", age: 5, adoptable: false)}
+  let!(:pet_2) {shelter_1.pets.create!(name: "Clawdia", breed: "shorthair", age: 3, adoptable: true)}
+  let!(:pet_3) {shelter_3.pets.create!(name: "Lucille Bald", breed: "sphynx", age: 8, adoptable: true)}
+  let!(:pet_4) {shelter_1.pets.create!(name: "Ann", breed: "ragdoll", age: 5, adoptable: true)}
+  let!(:pet_5) {shelter_2.pets.create!(adoptable: true, age: 2, breed: "dalmatian", name: "Pongo")}
+  let!(:pet_6) {shelter_3.pets.create!(adoptable: true, age: 1, breed: "dalmatian", name: "Pongo II")}
+
+  let!(:application_1) {Application.create!(name: "Sally", street_address: "112 W 9th St.", city: "Kansas City", state: "MO", zip_code: "64105", description: "I love animals. Please let me have one.", status: "pending")}
+  let!(:application_2) {Application.create!(name: "Marcus", street_address: "100 Hennepin Ave.", city: "Minneapolis", state: "MN", zip_code: "55401", description: "Dogs are the best. Please let me have one.", status: "pending")}
+
+  let!(:application_pet_1) {ApplicationPet.create!(pet: pet_1, application: application_1)}
+  let!(:application_pet_2) {ApplicationPet.create!(pet: pet_5, application: application_2)}
+
   describe "relationships" do
     it { should have_many(:pets) }
```

Comment:
Because you added the additional association to applications, remember to add that relationship test here:
`it { should have_many(:applications).through(:pets) }`